Classes/DoorbellDialog.m

```diff
@@ -296,8 +300,16 @@ - (void)createBoxSubviews
 
     _bodyView = [[UITextView alloc] initWithFrame:CGRectMake(10.0f, 45.0f, 280.0f, 100)];
     _bodyView.delegate = self;
-    _bodyView.backgroundColor = [UIColor systemBackgroundColor];
-//    _bodyView.textColor = [UIColor darkTextColor];
+    if (@available(iOS 13.0, *)) {
+        _bodyView.backgroundColor = [UIColor systemBackgroundColor];
+    } else {
+        _bodyView.backgroundColor = [UIColor whiteColor];
+    }
+    if (@available(iOS 13.0, *)) {
+        //    _bodyView.textColor = [UIColor darkTextColor];
+    } else {
+        _bodyView.textColor = [UIColor darkTextColor];
+    }
```

Comment:
This was only commented out once dark mode support was added, so I tried to keep things the same.

Reply:
Yep thanks! Needed to comment it out, otherwise the text color stayed dark even in dark mode and could not be read!